internal/config/config_type.go

```diff
@@ -12,9 +12,11 @@ const defaultGitProtocol = "https"
 
 // This interface describes interacting with some persistent configuration for gh.
 type Config interface {
+	Root() *yaml.Node
```

Comment:
Does Root need to be a part of the public interface? Right now it only seems to be accessed from within `fileConfig`.

Reply:
ah it's vestigial, good catch